python/lsst/fgcmcal/fgcmMakeLut.py

```diff
@@ -98,6 +99,21 @@ class FgcmMakeLutConnections(pipeBase.PipelineTaskConnections,
         dimensions=("instrument",),
     )
 
+    fgcmStandardAtmosphere = connectionTypes.Output(
```

Comment:
These are not calibration products because they are assumed to be valid for all data, correct?

Reply:
This is an interesting question.  They will be associated with the look-up table used which defines these standard passbands, and that is also not a calibration product because ... reasons?  In any event, you can reconfigure the look-up table construction, and choose a different standard airmass (for example), and this doesn't really have a "time" validity range, it just is a processing choice.